src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/MicrosoftQualityGuidelinesAnalyzersResources.resx

```diff
@@ -231,4 +231,7 @@
   <data name="AvoidDuplicateElementInitializationTitle" xml:space="preserve">
     <value>Do not duplicate indexed element initializations</value>
   </data>
+  <data name="MarkMembersAsStaticCodeFix" xml:space="preserve">
+    <value>Make static (Shared in VB)</value>
```

Comment:
Can we have separate strings for C# and VB? This might be confusing.

Reply:
I agree. The current terminology of the analyzer saying "(Shared in Visual Basic)" is very odd. Why not just have separate strings?

Reply:
Certainly, let me split.

Reply:
Actually, we don't seem to have resource assemblies for C#/VB projects, and it seems an overkill to introduce them just for this one case - let me just switch the code fix title to `Make static`, which seems pretty clear.